src/js/modules/RoutingApp.js

```diff
@@ -180,8 +180,8 @@ class RoutingApp extends PureComponent {
               />
               <Route 
```

Comment:
Add an alias for recruiting otherwise old links will break

Reply:
I don't think anyone posted the old link online anywhere. The reason I'm changing it is because the EICs wanted that URL to post

Reply:
Okay